src/app/components/cds/inputs/TextField.test.js

```diff
@@ -36,13 +35,13 @@ describe('<TextField />', () => {
 
   it('should render icons', () => {
     const textField = shallow(<TextField
-      iconLeft={<VolumeUpIcon />}
-      iconRight={<VolumeOffIcon />}
+      iconLeft={<VolumeUpIcon className="icon__vol-up"/>}
+      iconRight={<VolumeOffIcon className="icon__vol-off"/>}
```

Comment:
@brianfleming Basically the fix for the unit tests thing is to search for icons in test via class name rather than component name. This is because our custom icons are not actually rendered with a component name when we mock them, they are just raw SVG! So when we switch from a React component (such as a material-ui icon) to a bare SVG (like our custom icons), we can no longer use Jest's "find the component" shortcut selector.

Reply:
ah, thats good to know, thanks @dariusk . will help me diagnose in the future